app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -430,22 +430,22 @@ def process_vm_physical_dvd_drive(dvd)
     end
 
     def process_iso_image(vm)
-      path = vm[:DVDs][:Props][:SharePath]
-      size = vm[:DVDs][:Props][:Size]
-      uid  = vm[:DVDs][:Props][:ID]
-      name = vm[:DVDs][:Props][:Name]
+      # Some hoops for backwards compatibility
+      vm[:DVDs] = [vm[:DVDs]] unless vm[:DVDs].kind_of?(Array)
```

Comment:
@djberg96 
You are forcing vm[:DVDs] to be an array in the powershell script, regardless of the number of DVDs, so i do not think you need this line.